Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -706,6 +735,10 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
     m_ws->mutableRun().addProperty("run_start", run_start.toISO8601String(),
                                    true);
   }
+  // Make a pointer to a const period log, as it should no longer be changed
+  auto periodLog =
```

Comment:
I was thinking that this could move into `runLoadNexusLogs`? In addition, I think if we added a constructor to `TimeSeriesProperty`:

```c++
TimeSeriesProperty(const std::string &name, const std::vector<Types::Core::DateAndTime> &times,
                                const std::vector<TYPE> &values) : TimeSeriesProperty(name) {
  addValues(times, values);
}

then we could delay creating the object until we have the values and then avoid this copy? What do you think?

Reply:
Good idea. I will look at sorting this in `runLoadNexusLogs`